libs/utils/source/ftrack_utils/paths/__init__.py

```diff
@@ -56,3 +59,59 @@ def get_temp_path(filename_extension=None):
         os.makedirs(os.path.dirname(result))
 
     return result
+
+
+def check_image_sequence(path, with_members=False):
```

Comment:
Can we use clique or something else for this?

Reply:
Its removed.